### ISSUE TITLE
2.1.0 needs to be merged in master

### ISSUE THREAD

**Issue by remram44** [UV-CDAT/uvcdat#989]
git-describe currently gives out:
```
$ git describe --always --tags
2.0.0-616-g804abe8
```
As far as I can see, the commit tagged 2.1.0 is the only one not in master's history:
```
$ git log --oneline --graph --decorate origin/master..2.1.0
* fdb9e30 (tag: uvcdat-2.1.0, tag: 2.1.0, origin/release-2.1.0, origin/release)
```

**Comment by doutriaux1** [comment]
can you  merge a tag via PR?

**Comment by doutriaux1** [comment]
```doutriaux1@omar:[cdutil]:[master]:[27343]> git merge --no-ff 2.1.0
Already up-to-date.
doutriaux1@omar:[cdutil]:[master]:[27344]> 
```

**Comment by doutriaux1** [comment]
I agree this describe thing is annoying, we usually create the tag point on master at time of release, but this time a bunch of things have been merged into master before the final tag point (on release branch) was applied. I'm not positive how on how to bring that tag point back into master now...

**Comment by remram44** [comment]
> can you merge a tag via PR?

It is the branch release-2.1.0: https://github.com/UV-CDAT/uvcdat/compare/master...release-2.1.0?expand=1

**Comment by remram44** [comment]
Fixed with #990.
